core/indigo-core/graph/graph.h

```diff
@@ -24,6 +24,9 @@
 #include "base_cpp/non_copyable.h"
 #include "base_cpp/obj_array.h"
 #include "base_cpp/obj_pool.h"
+#include <list>
+#include <memory>
```

Comment:
Standard library headers should be included before local headers.

Reply:
BTW, I don't see any usage of list or memory here.

Reply:
fixed